main.js

```diff
@@ -129,7 +129,7 @@ function LinkBox()
 		title: 'Join',
 		message: 'Would you like to spectate or join?',
```

Comment:
Change from
Would you like to spectate or join?
to
Would you like to spectate or play?